code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -155,6 +156,7 @@
 	hair_color = "mutcolor"
 	hair_alpha = 150
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
+	mutanteyes = /obj/item/organ/internal/eyes
```

Comment:
i think you want to use null here and the other one

Reply:
Setting this to null actually runtimes - this is what the default value of mutanteyes is. As best I can tell, mutanteyes is _always_ what is used to determine what someone's eyes should be, even for humans. Kind of a badly-named var, but eh. Because the base jellypeople should use three eyes but two of the subtypes should not, I have to set it back to the default like this.

Reply:
Not ideal but it just be like that unless someone wants to throw around the whole jelly parent type